misc/make-deps.sh

```diff
@@ -39,7 +39,8 @@ if [ ! -z "$APT" ]; then
 fi
 
 if [ ! -z "$BREW" ]; then
-	$BREW install libvirt || true
+	# coreutils contains gtimeout
+	$BREW install libvirt augeas coreutils|| true
```

Comment:
i think a space before the `||` would be welcome, but i'd certainly merge without this, since it's very nitpicky

Reply:
You are absolutely right, a space is more than welcome.